charts/newrelic-infrastructure/templates/kubelet/_agent-config_helper.tpl

```diff
@@ -6,6 +6,10 @@ http_server_enabled: true
 http_server_port: 8003
 features:
   docker_enabled: false
+{{- with .Values.kubelet.passthroughEnvironment }}
+passthrough_environment:
+  {{- toYaml . | nindent 2 }}
+{{- end }}
```

Comment:
What is the downside of specifying `passthrough_environment` in `agentConfig` directly?

Reply:
I get the side effect but, does it matter? Agents in CP/KSM scrapers run in forward mode and therefore cannot run integrations.

Reply:
I did not think about agents running in `forward_only` mode.

Anyway, I have done it in a generic way and added documentation for that.